docs/source/getting-started/quickstart-2/index.rst

```diff
@@ -76,68 +79,75 @@ Now load the dataset and split it into training, validation, and test sets.
+            [
+                keras.Input([train_x.shape[1]]),
+                keras.layers.Normalization(mean=np.mean(train_x), variance=np.var(train_x)),
+                keras.layers.Dense(64, activation="relu"),
+                keras.layers.Dense(1),
+            ]
         )
-        model.add(Dense(64, activation="relu", input_shape=(train_x.shape[1],)))
-        model.add(Dense(1))
 
         # Compile model
         model.compile(
-            optimizer=SGD(lr=params["lr"], momentum=params["momentum"]),
+            optimizer=keras.optimizers.SGD(
+                learning_rate=params["lr"], momentum=params["momentum"]
+            ),
             loss="mean_squared_error",
+            metrics=[keras.metrics.RootMeanSquaredError()],
         )
 
         # Train model with MLflow tracking
         with mlflow.start_run(nested=True):
-            # Fit model
             model.fit(
                 train_x,
                 train_y,
                 validation_data=(valid_x, valid_y),
                 epochs=epochs,
-                verbose=0,
+                batch_size=64,
             )
-
             # Evaluate the model
-            predicted_qualities = model.predict(test_x)
-            rmse = np.sqrt(mean_squared_error(test_y, predicted_qualities))
+            eval_result = model.evaluate(test_x, test_y, batch_size=64)
+            eval_rmse = eval_result[1]
 
             # Log parameters and results
             mlflow.log_params(params)
-            mlflow.log_metric("rmse", rmse)
+            mlflow.log_metric("eval_rmse", eval_rmse)
 
             # Log model
             mlflow.tensorflow.log_model(model, "model", signature=signature)
 
-            return {"loss": rmse, "status": STATUS_OK, "model": model}
+            return {"eval_rmse": eval_rmse, "status": STATUS_OK, "model": model}
+
 
-The ``objective`` function takes in the hyperparameters and returns the results of the ``train_model`` function for that set of hyperparameters.
+The ``objective`` function takes in the hyperparameters and returns the results of the ``train_model`` 
+function for that set of hyperparameters.
 
 .. code-block:: python
 
     def objective(params):
         # MLflow will track the parameters and results for each run
         result = train_model(
             params,
+            epochs=3,
             train_x=train_x,
             train_y=train_y,
             valid_x=valid_x,
             valid_y=valid_y,
             test_x=test_x,
             test_y=test_y,
-            epochs=32,  # Or any other number of epochs
         )
         return result
 
-Next, we will define the search space for Hyperopt. In this case, we want to try different values of ``learning-rate`` and ``momentum``. 
+Next, we will define the search space for Hyperopt. In this case, we want to try different values of 
+``learning-rate`` and ``momentum``. Hyperopt randomly picks a value from the range we define for ``lr`` and ``momentum``.
```

Comment:
It initially does a random selection, then does a bayesian selection criterion for subsequent iterations after the initial results from the minimum random population selection return. 

We could add in something like this, maybe?

Hyperopt begins its optimization process by selecting an initial set of hyperparameters, typically chosen at random or based on a specified domain space. This domain space defines the range and distribution of possible values for each hyperparameter. After evaluating the initial set, Hyperopt uses the results to update its probabilistic model, guiding the selection of subsequent hyperparameter sets in a more informed manner, aiming to converge towards the optimal solution.